src/test/unit/algorithms/ConnectionsTest.cpp

```diff
@@ -343,6 +361,175 @@ TEST(ConnectionsTest, testComputeActivity) {
+      {0.22, 0.1001, 0.15, 0.051, 0.07},   // increment four times
+      {0.101, 0.101, 0.101, 0.101, 0.101}, // increment 9 times
+      {0.17, 0.106, 0, 0, 0.128},          // increment 5 times
+      {0, 0.101, 0.11, 0.18, 0}};          // increment 4 times
+
+  UInt trueConnectedCount[7] = {3, 3, 4, 3, 5, 3, 3};
+
+  for (UInt i = 0; i < numColumns; i++) {
+    vector<Real> perm;
+    vector<UInt> potential;
+    perm.assign(&permArr[i][0], &permArr[i][numInputs]);
+    for (UInt j = 0; j < numInputs; j++) {
+      if (potentialArr[i][j] > 0) {
+        potential.push_back(j);
+      }
+    }
+    UInt connected = sp.raisePermanencesToThreshold_(perm, potential);
+    ASSERT_TRUE(check_vector_eq(truePerm[i], perm));
+    ASSERT_TRUE(connected == trueConnectedCount[i]);
+  }
+  */
+}
+
+
+TEST(ConnectionsTest, testBumpUpWeakColumns) {
+  FAIL(); /*
+  SpatialPooler sp;
+  UInt numInputs = 8;
+  UInt numColumns = 5;
+  setup(sp, numInputs, numColumns);
+  sp.setSynPermBelowStimulusInc(0.01);
+  sp.setSynPermTrimThreshold(0.05);
+  Real overlapDutyCyclesArr[] = {0, 0.009, 0.1, 0.001, 0.002};
+  sp.setOverlapDutyCycles(overlapDutyCyclesArr);
+  Real minOverlapDutyCyclesArr[] = {0.01, 0.01, 0.01, 0.01, 0.01};
+  sp.setMinOverlapDutyCycles(minOverlapDutyCyclesArr);
+
+  UInt potentialArr[5][8] = {{1, 1, 1, 1, 0, 0, 0, 0},
+                             {1, 0, 0, 0, 1, 1, 0, 1},
+                             {0, 0, 1, 0, 1, 1, 1, 0},
+                             {1, 1, 1, 0, 0, 0, 1, 0},
+                             {1, 1, 1, 1, 1, 1, 1, 1}};
+
+  Real permArr[5][8] = {
+      {0.200, 0.120, 0.090, 0.040, 0.000, 0.000, 0.000, 0.000},
+      {0.150, 0.000, 0.000, 0.000, 0.180, 0.120, 0.000, 0.450},
+      {0.000, 0.000, 0.074, 0.000, 0.062, 0.054, 0.110, 0.000},
+      {0.051, 0.000, 0.000, 0.000, 0.000, 0.000, 0.178, 0.000},
+      {0.100, 0.738, 0.085, 0.002, 0.052, 0.008, 0.208, 0.034}};
+
+  Real truePermArr[5][8] = {
+      {0.210, 0.130, 0.100, 0.000, 0.000, 0.000, 0.000, 0.000},
+      //  Inc    Inc    Inc    Trim    -     -     -    -
+      {0.160, 0.000, 0.000, 0.000, 0.190, 0.130, 0.000, 0.460},
+      //  Inc   -     -    -     Inc   Inc    -     Inc
+      {0.000, 0.000, 0.074, 0.000, 0.062, 0.054, 0.110, 0.000}, // unchanged
+      //  -    -     -    -     -    -     -    -
+      {0.061, 0.000, 0.000, 0.000, 0.000, 0.000, 0.188, 0.000},
+      //   Inc   Trim    Trim    -     -      -     Inc     -
+      {0.110, 0.748, 0.095, 0.000, 0.062, 0.000, 0.218, 0.000}};
+
+  for (UInt i = 0; i < numColumns; i++) {
+    sp.setPotential(i, potentialArr[i]);
+    sp.setPermanence(i, permArr[i]);
+    Real perm[8];
+    sp.getPermanence(i, perm);
+  }
+
+  sp.bumpUpWeakColumns_();
+
+  for (UInt i = 0; i < numColumns; i++) {
+    Real perm[8];
+    sp.getPermanence(i, perm);
+    ASSERT_TRUE(check_vector_eq(truePermArr[i], perm, numInputs));
+  }
+  */
+}
+
+
+TEST(ConnectionsTest, testConnectedCount) {
+  FAIL();
```

Comment:
TODO

Reply:
ConnectedCount is an SP method, not a Connections method. I meant to move it from SP to Connections but I haven't made this happen yet and probably will not for this PR. So I will remove this test-case stub and use whatever tests the SP had previously for ConnectedCount (which is only test case `testUpdatePermanencesForColumn`, now renamed to `testSetPermanence`).